### DISCUSSION TITLE
AIR: compile-adt-64 problem

### DISCUSSION THREAD

**Discussion by ZhorovEgor** [airsdk/Adobe-Runtime-Support#3105] [Q&A]
Howdy! When i publish app, i see this problem what is it?:
![Screenshot_1](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/e2038f1b-8b85-4ec6-95b0-05a0bdaa0ce6)
After click OK, i see it:
![Screenshot_2](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/63f7f590-208c-48c0-8758-d55b593012ec)

**Comment by FliplineStudios** [comment]
If you're on a Mac, you may have to remove the SDK from quarantine -- Section 1.3 in the Release Notes PDF has more info on the command to use:

`$ xattr -d -r com.apple.quarantine /path/to/SDK`

**Reply by ajwfrost** [reply]
... I'm really hoping that we don't have to notarise it too! Will see if we need to do any macos updates and check on the latest version here.
Just checking @ZhorovEgor  - what version of macOS are you currently using there, and is it an "Apple Silicon" one (M1/M2/M3) or an Intel one?

thanks

**Reply by ZhorovEgor** [reply]
Actualy i'm not Mac user. I bought Mac server and try use him. Now i see i cant instal Xcode becouse OS version too small (12.7) Xcode use 13.5 or more. I make update server OS on this moment and after i will tell you MacOS version.

**Reply by ZhorovEgor** [reply]
![Screenshot_1](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/e7545f31-52bc-4b9b-b382-a003d67ac8f8)

**Reply by ajwfrost** [reply]
Okay so that shouldn't be a notarisation requirement then afaik, for Monterey and an x86_64 binary. Are you able to open the terminal window and remove the quarantine per the suggestion from @FliplineStudios ?

thanks

**Reply by ajwfrost** [reply]
Just to confirm, we tried this - with the SDK downloaded/unzipped from Safari - and got the same error result at first (also similar warnings about idb, but we could ignore those ones). Having then removed the quarantine extended attributes, it worked fine...

Best is to use AIR SDK Manager as this doesn't end up with you getting the quarantine bit set in the first place!

thanks

**Reply by raresn** [reply]
we run the $ xattr -d -r com.apple.quarantine /path/to/SDK command each time we download a new zip :)
